### ISSUE TITLE
Expected URL encoded form data, got JSON.

### ISSUE THREAD

**Issue by dmalis** [mollie/mollie-api-python#67]
Env:
PY@APPENGINE, Flask=1.0.2, requests=2.19.1, mollie=2.0.0

Basic payment creation
```
    mollie_client = Client()
    mollie_client.set_api_key('test_V........')
    payment = mollie_client.payments.create({
        'amount': {
            'currency': 'EUR',
            'value': '10.00'
        },
        'description': 'My first API payment',
        'redirectUrl': 'https://webshop.example.org/order/12345/',
        'webhookUrl': 'https://webshop.example.org/mollie-webhook/',
    })
```

fails with

```
>> File "/Users/damjan/development/bumpyshop/src/lib/mollie/api/resources/base.py", line 54, in create
>> return self.rest_create(data, params)
>> File "/Users/damjan/development/bumpyshop/src/lib/mollie/api/resources/base.py", line 26, in rest_create
>> result = self.perform_api_call(self.REST_CREATE, path, data, params)
>> File "/Users/damjan/development/bumpyshop/src/lib/mollie/api/resources/base.py", line 87, in perform_api_call
>> raise ResponseError.factory(result)
>> ResponseError: Expected URL encoded form data, got JSON. Set the "Content-Type" header to "application/json" when sending JSON encoded data.
```

**Comment by mvdpanne** [comment]
Hey dmalis,

Thanks for reporting your issue. The error message you got only gets returned if `application/x-www-form-urlencoded` is passed as `Content-Type` request header.

The API client does not enforce this; it does not send a `Content-Type` header at all. The only thing I can think of is that you probably have set this header globally for the `requests` package, somehow.

But, since it is easy to let the API client override it, tomorrow I'll make changes to the client to ensure that the `Content-Type` header is correctly set by the client itself.

**Comment by dmalis** [comment]
Maikel thank you for prompt response. I've checked and I'm certain I never set Content-Type header myself. 

I guess this is appengine's way of filling in the missing header/s.

From: https://cloud.google.com/appengine/docs/standard/python/how-requests-are-handled
Note: In the absence of a Content-Type header provided by your application, a default Content-Type: text/html header will be set, but the automatic gzip compression will not be applied.

Thank you for fixing this, as I was looking for quite some time to start using V2 library instead of "raw api".

**Comment by mvdpanne** [comment]
Hey dmalis,

I just released API client v2.0.1. This new version should fix your problem. It would be great if you could let me know whether it works.

**Comment by dmalis** [comment]
Perfect, it works. Thank you Maikel.
